bson/raw_bson.py

```diff
@@ -13,6 +13,46 @@
 # limitations under the License.
 
 """Tools for representing raw BSON documents.
+
+Inserting and Retrieving RawBSONDocuments
+=========================================
+
+Example: Moving a document between different databases/collections
+.. code-block:: python
+
+  import bsonjs
```

Comment:
We shouldn't be using `bsonjs` here because we don't know if the user is familiar with that package. It is also not needed for this particular usecase.
To be clear, `bsonjs` is used to directly convert between JSON (which is a string) and BSON (which is binary `bytes`). It saves us the trouble of having to convert everything to Python dicts when converting BSON data into JSON. E.g. the following snippet would retrieve a document and print it as JSON:
```
doc = db.coll.find_one()
print(json.dumps(doc))
```
However, in doing so we actually end up doing 2x the work we need to. `doc = db.coll.find_one()` gets the document in BSON format from the server and converts it to a Python dict. `json.dumps(doc)` converts that Python dict to a JSON string. With `bsonjs.dumps` we can directly convert BSON bytes to a JSON string hence saving us CPU time. This is the main usecase for `bsonjs` as people often want to server query results as JSON. However, this is not needed to demonstrate usage of `RawBSONDocument`.

Reply:
rm'd